src/compiler/types.ts

```diff
@@ -316,6 +316,7 @@ namespace ts {
         // Property assignments
         PropertyAssignment,
         ShorthandPropertyAssignment,
+        SpreadObjectLiteralAssignment,

```

Comment:
There's already a `SpreadElementExpression` used for array spread elements - is it possible we can use that?